### DISCUSSION TITLE
example on how between is used

### DISCUSSION THREAD

**Discussion by terrymight** [abbasudo/laravel-purity#73] [Q&A] [answered]
am trying to filter between dates but I can`t figure it out. 

`http://localhost:8000/api/v1/list?filters[created_at][$between]=2024-06-03&[$and]=2024-06-05`
Thanks

**Comment by terrymight** [comment, marked as answer]
Thank you I have figured it out, this is how I was able to do it

`http://localhost:8000/api/v1/list?filters[created_at][$between][0]=2024-06-03&filters[created_at][$between][1]=2024-06-05`

**Comment by abbasudo** [comment]
@terrymight Great. I'll include more usage examples in the docutmets.
